Animal Competition/Java/Wolf.java

```diff
@@ -20,19 +20,19 @@ public boolean hasTail() {
     return true;
   }
 // Animals abilities are rated on a scale from 0-100 where 0 is they can't do it and 100 being that they excell at it.
```

Comment:
You don't need these redundant comments in all the sub-classes, the original comment on the parent class or interface will suffice.